endpoint.go

```diff
@@ -723,14 +723,6 @@ func (ep *endpoint) sbLeave(sb *sandbox, force bool, options ...EndpointOption)
 }
 
 func (n *network) validateForceDelete(locator string) error {
-	if n.Scope() == datastore.LocalScope {
-		return nil
-	}
-
-	if locator == "" {
-		return fmt.Errorf("invalid endpoint locator identifier")
-	}
-
```

Comment:
This is my theory:
In the initial patch, there was additional code doing an extra validation check:
```
if n.getController().isNodeAlive(locator) {
		return fmt.Errorf("host alive : %s", locator)
}
```
In order for the above code to do a meaningful check, locator had to be non-empty.

But PR review comments asked to remove the restriction on `isNodeAlive()`.
But the check on non-empty locator stayed...